src/router.js

```diff
@@ -6,6 +6,27 @@ const users = [
     {id:1, username:'user1', email:'user1@gmail.com'},
     {id:2, username:'user2', email:'user2@gmail.com'}]
 
+
+router.post('/profile', (req, res, next) => {
+    const {username, email} = req.body
+    const id = users.length
+    const newUser = {id, username, email}
+    console.log(newUser);
+    users.push(newUser)
+    res.json(newUser)
+})
+
+router.put('/profile/:id', (req, res, next) => {
+    const {id} = req.params
+    const {username, email} = req.body
+    const currentIndex = users.findIndex(u=>u.id===Number(id))
```

Comment:
Could you please follow our coding style, for example here you need to add spaces between `===`